doc-generator/src/main/scala/codacy/cppcheck/DocGenerator.scala

```diff
@@ -310,6 +466,8 @@ object DocGenerator {
     Json.parse(Json.toJson(misraPatterns).toString).as[JsArray]
   }
 
+  private def getMisraTitle(rule: String): String = titles.getOrElse(rule, s"MISRA $rule rule")
+
```

Comment:
I don't know if it's correct to use `getOrElse` here. Because if we update the tool, we want to ensure that all the misra rules have a proper title. This way it's easy to overlook a missing title in the `Map`

Reply:
I am not sure that enforcing it to have all titles would bring also negative impacts, but I agree with you that we could easily overlook if we use `getOrElse`. I can use `get` instead